packages/feedback/src/core/components/Actor.css.ts

```diff
@@ -56,6 +56,12 @@ export function createActorStyles(): HTMLStyleElement {
 .widget__actor__text {
 }
 
+@media (max-width: 600px) {
```

Comment:
Suggestions for another number for viewport boundary is welcome!

Reply:
looks like most mobile devices have widths < 600
only ipads have viewports of 700+ which means ipad users would see the big button, which is probably fine!

Reply:
https://www.icwebdesign.co.uk/common-viewport-sizes looking at this very non-sus website